CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 
+- Added support for Kupo service in nix/runtime.nix ([#1164](https://github.com/Plutonomicon/cardano-transaction-lib/issues/1164)) NOTE: Does not fully resolve that issue, still need to pull in Kupo as a binary
```

Comment:
Please add to the end of the changelog. Also, no need to add at all, since the issue is not resolved.